hangman/hangman.py

```diff
@@ -36,6 +38,11 @@ def hangman():
             for val in dic_list:
                 print(val + ": {} ".format(dic_list[val]))
             break
+        elif count == times:
+            print("The END")
```

Comment:
Na wa oo, carry your users along na. Why will I be playing game and suddenly see "The end" with no reason why the game is over?

Reply:
Lol, sorry.